proto/umee/auction/v1/tx.proto

```diff
@@ -44,7 +43,7 @@ message MsgRewardsBid {
   // the current auction ID being bid on. Fails if the ID is not an ID of the current auction.
   uint32 id = 2;
   // amount of the bid in the base tokens
-  cosmos.base.v1beta1.Coin bid_amount = 3 [(gogoproto.nullable) = false];
+  string amount = 3 [(gogoproto.customtype) = "cosmossdk.io/math.Int", (gogoproto.nullable) = false];
```

Comment:
```suggestion
 cosmos.base.v1beta1.Coin bid_amount = 3 [(gogoproto.nullable) = false];
```
It is better to have Coin format because we  can validate the denom